generator/src/test/java/com/scottlogic/deg/generator/cucumber/InSet.feature

```diff
@@ -1355,10 +1355,9 @@ Scenario: 'InSet' with a non contradicting not 'granularTo' is successful
       | null |
       | 1.1  |
 
+@ignore #issue #824 - 2.0 should be generated with a granularity of 1
 Scenario: Not 'inSet' with a non contradicting 'granularTo' is successful
   Given there is a field foo
-    And foo is anything but in set:
```

Comment:
Why is this being removed and the scenario ignored. Does removing the set invalidate the test?

2.0 is an integer, I'd not expect to see `.0` except where a `formattedAs` constraint enforced it.

Reply:
The test was previously invalid. Having an `inSet` with a value of `1.1` and a granularity of `1` implies that `1.1` should not be present in the output. This test now fails due to `2.0` not being recognised as an integer. Even with `2` expected in the output the test still fails.